configure.ac

```diff
@@ -48,6 +48,8 @@ AS_CASE([${RKT_STAGE1_USR_FROM}],
                  RKT_REQ_PROG([LIBTOOLIZE],[libtoolize],[libtoolize])],
         [host],
                 [],
+        [kvm],
+                [],
```

Comment:
Consider putting `AC_MSG_WARN([* kvm is an experimental stage1 implementation, not all features work])` between square brackets.